app/src/main/java/org/jellyfin/androidtv/util/BlurHashUtils.kt

```diff
@@ -0,0 +1,27 @@
+@file:JvmName("BlurHashUtils")
+
+package org.jellyfin.androidtv.util
+
+import android.graphics.Bitmap
+import androidx.lifecycle.LifecycleOwner
+import androidx.lifecycle.lifecycleScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.withContext
+
+fun createBlurHashDrawable(
+	lifecycleOwner: LifecycleOwner,
+	blurHash: String?,
+	width: Int,
+	height: Int,
+	callback: (bitmap: Bitmap?) -> Unit,
+) = lifecycleOwner.lifecycleScope.launch {
```

Comment:
Could really simplify this part now (replaced the GlideExtensions + BlurHashService code)